app/controllers/catalog_controller.rb

```diff
@@ -191,7 +191,7 @@ def exclude_unwanted_records(solr_parameters,user_parameters)
     
     config.add_search_field ('Keyword (Title, Name/Creator, Topic, Notes etc.)') do |field|
 	  #field.solr_parameters = { :'qf' => 'all_fields_tesim' }
-	  field.solr_parameters = { :'qf' => 'title_tesim^99 name_tesim^20 subject_tesim^10 scopeContentNote_tesim^6 all_fields_tesim fulltext_tesim id' }
+	  field.solr_parameters = { :'qf' => 'title_tesim^99 name_tesim^20 topic_tesim^10 scopeContentNote_tesim^6 all_fields_tesim fulltext_tesim id' }
```

Comment:
@lsitu If use subject_tesim, the topics didn't show up in the search result page that's what i found the problem source described in the ticket:  See http://library.ucsd.edu/dc/search?f%5Bcollection_sim%5D%5B%5D=Dr.+Seuss+Political+Cartoons&id=bb65202085

Look at this line of code: config.add_search_field Keyword (Title, Name/Creator, Topic, Notes etc.),  Should we explicitly add this four solr_parameters only?